packages/core/README.md

```diff
@@ -1,5 +1,15 @@
 # Vibe Design System - Core
 
+# ⚠️ Deprecation Notice
```

Comment:
@YossiSaadi It seems like you're deprecating this package as it's on the top and highlighted. Also we referred to it in the main readme also and it has a link to npm and the new package is mentioned in the getting started

Reply:
@talkor What would you like to write there then?

Reply:
How about deprecating the npm package instead? It will be shown when you install it which will be more useful IMO

Reply:
You said it is too aggressive, that's why I didn't just do it 🙄🫢

Reply:
I said "not at the moment" haha